Frends.Zip.CreateArchive/Frends.Zip.CreateArchive.Tests/UnitTests.cs

```diff
@@ -5,316 +5,288 @@
+        Assert.AreEqual("zip_test_(1).zip", Path.GetFileName(result2.Path));
+        Assert.AreEqual("zip_test_(2).zip", Path.GetFileName(result3.Path));
+    }
 
-            var zipFiles = Directory.GetFiles(_destination.Directory, "*.zip");
+    [Test]
+    public void ZipFile_Exists_Appends_NewFiles()
+    {
+        var result = ExecuteCreateArchive();
+        Assert.AreEqual(2, result.FileCount);
+        _options.DestinationFileExistsAction = FileExistAction.Append;
+        _destination.RenameDuplicateFiles = true;
+        var appendResult = ExecuteCreateArchive();
+        Assert.AreEqual(4, appendResult.FileCount);
+    }
 
-            Assert.AreEqual(3, zipFiles.Count());
-            Assert.AreEqual("zip_test.zip", Path.GetFileName(result1.Path));
-            Assert.AreEqual("zip_test_(1).zip", Path.GetFileName(result2.Path));
-            Assert.AreEqual("zip_test_(2).zip", Path.GetFileName(result3.Path));
-        }
+    [Test]
+    public void ZipFiles_WithPassword_NeedsPasword_For_Extraction()
+    {
+        _destination.Password = "password";
+        var result = ExecuteCreateArchive();
+        var zipFileName = result.Path;
+        var extractPath = Path.Combine(_dirOut, "extracted");
 
-        [Test]
-        public void ZipFile_Exists_Appends_NewFiles()
+        using (var zip = ZipFile.Read(zipFileName))
         {
-            var result = ExecuteCreateArchive();
-            Assert.AreEqual(2, result.FileCount);
-            _options.DestinationFileExistsAction = FileExistAction.Append;
-            _destination.RenameDuplicateFiles = true;
-            var appendResult = ExecuteCreateArchive();
-            Assert.AreEqual(4, appendResult.FileCount);
-
+            Assert.Throws<BadPasswordException>(() => zip.ExtractAll(extractPath));
+            zip.Password = _destination.Password;
+            zip.ExtractAll(extractPath);
         }
 
-        [Test]
-        public void ZipFiles_WithPassword_NeedsPasword_For_Extraction()
-        {
-            _destination.Password = "password";
-            var result = ExecuteCreateArchive();
-            var zipFileName = result.Path;
-            var extractPath = Path.Combine(_dirOut, "extracted");
-
-            using (var zip = ZipFile.Read(zipFileName))
-            {
-                Assert.Throws<BadPasswordException>(() => zip.ExtractAll(extractPath));
-
-                zip.Password = _destination.Password;
-                zip.ExtractAll(extractPath);
-            }
-
-            Assert.AreEqual(_zipFileName, Path.GetFileName(result.Path));
-            Assert.IsTrue(Directory.Exists(extractPath));
-            Assert.AreEqual(result.FileCount, Directory.GetFiles(extractPath, "*").Count());
-        }
+        Assert.AreEqual(_zipFileName, Path.GetFileName(result.Path));
+        Assert.IsTrue(Directory.Exists(extractPath));
+        Assert.AreEqual(result.FileCount, Directory.GetFiles(extractPath, "*").Count());
```

Comment:
`Directory.GetFiles()` returns an array, not List, so you should use `.Length` instead of `.Count`.